### DISCUSSION TITLE
Playgroundをコミットしたいです

### DISCUSSION THREAD

**Discussion by KOTARO1209** [lit-iphone-course/lit-iphone-course#101] [Question-original]
Playgroundはどのようにしてギットハブに上げるのでしょうか？

**Comment by txxxxc** [comment]
普通のプロジェクトと同様に、Source ControlからNew Git Repositoryでリポジトリを作ってみてくださいー！
![image](https://user-images.githubusercontent.com/22112440/215789974-fb15ab75-31b6-45f6-97a8-e5c7eaec70d6.png)

**Reply by KOTARO1209** [reply]
名前が「Hokan」になっておらず、うまくいかないです🙇

![image](https://user-images.githubusercontent.com/75009903/215791667-d3f1ae36-f5e8-4f94-bcb4-ad9fa4d3faab.png)
![image](https://user-images.githubusercontent.com/75009903/215791817-623377de-b67c-4175-90e3-e66df4dddb5b.png)

**Comment by txxxxc** [comment]
既にリポジトリが作られてるっぽい大丈夫そうです！名前に関してはGitHubの方から変えられると思います！
(補完の教科書に関してはGitHubに上げなくても大丈夫なので、一旦このエラー飛ばしてもらってもOKです！)
